src/txn/txn_ckpt.c

```diff
@@ -639,6 +643,31 @@ __checkpoint_prepare(WT_SESSION_IMPL *session, bool *trackingp, const char *cfg[
     WT_ASSERT(session, session->ckpt_handle_next == 0);
     WT_WITH_TABLE_READ_LOCK(
       session, ret = __checkpoint_apply_all(session, cfg, __wt_checkpoint_get_handles));
+
+    /* Add the history store handle to the checkpoint handle queue. Currently recovery has
+     * an issue with this where the LAS file may not exist yet. As recovery runs checkpoint
+     * and it can run before the lookaside exists we ignore failures.
+     */
+    t_ret = ret;
+    if (session->ckpt_handle_next == 0 ||
+      !WT_IS_LAS(session->ckpt_handle[session->ckpt_handle_next - 1]))
+        WT_WITH_TABLE_READ_LOCK(session,
+          ret = __wt_schema_worker(session, WT_LAS_URI, __wt_checkpoint_get_handles, NULL, cfg, 0));
+    /* Ignore return value of schema worker call for time being. */
```

Comment:
Are you doing this just to avoid a compiler warning for ignoring the return value? Afaik, you can just do `(void)__wt_schema_worker(...)`. I've seen us doing this elsewhere in the code.

Reply:
Added WT_IGNORE_RET macro.